dashboard/config/levels/custom/music/musiclab_intro_change_sound.level

```diff
@@ -51,15 +54,45 @@
         {
           "conditions": [
             {
-              "name": "played_sounds_together",
-              "value": 1
+              "name": "played_sound_id",
+              "value": "beats/drum_beat_edm"
             }
           ],
-          "message": "Nice work!",
+          "message": "Nice work.  You successfully changed the sound.",
           "next": true
         },
         {
-          "message": "Please change the sound."
+          "conditions": [
+            {
+              "name": "played_sound_id",
+              "value": "beats/drum_beat_club"
+            }
+          ],
+          "message": "Nice work.  You successfully changed the sound.",
+          "next": true
+        },
+        {
+          "conditions": [
+            {
+              "name": "played_sound_id",
+              "value": "beats/drum_beat_knock"
+            }
+          ],
+          "message": "Nice work.  You successfully changed the sound.",
+          "next": true
+        },
+        {
+          "conditions": [
+            {
+              "name": "played_sound_id",
+              "value": "beats/drum_kit_pop"
+            }
+          ],
+          "message": "Nice work.  You successfully changed the sound.",
+          "next": true
+        },
```

Comment:
Ideally, we'd be able to have a single condition along the lines of "if played sound is not equal to X".  

But in our current architecture, we record successful conditions first in [`music/progress/MusicValidator`](https://github.com/code-dot-org/code-dot-org/blob/138fc9cf8989cf6a67cdd805e98a7a8847b03450/apps/src/music/progress/MusicValidator.ts), and later check the level's specified conditions against that previously-recorded list in [`lab2/progress/ConditionsChecker`](https://github.com/code-dot-org/code-dot-org/blob/138fc9cf8989cf6a67cdd805e98a7a8847b03450/apps/src/lab2/progress/ConditionsChecker.ts).  

To get that single condition, I think we'd need to adjust the architecture so that `music/progress/MusicValidator` knows about the level's specified conditions.